src/data/fintype/basic.lean

```diff
@@ -1318,6 +1318,14 @@ by rw [←e.equiv_of_fintype_self_embedding_to_embedding, univ_map_equiv_to_embe
 
 namespace fintype
 
+/-- Given `fintype α`, `equiv_set_on_fintype` is the equiv between `finset α` and `set α`. (All
+set on a finite type are finite.) -/
+@[simps] noncomputable def equiv_set_on_fintype [fintype α] : finset α ≃ set α :=
```

Comment:
I'd suggest `fintype.finset_equiv_set` for this rather than `fintype.equiv_set_on_fintype`.

Reply:
I'm matching the naming on `finsupp.equiv_fun_on_fintype`. I like your name better, but I'd rather rename that to `fintype.finsupp_equiv_fun` first.

Reply:
If you were to follow the naming on that `finsupp` definition I think you'd have `finset.equiv_set_on_fintype` instead.